src/desktop/tabletinput.cpp

```diff
@@ -218,24 +163,39 @@ void update(QApplication *app, const QSettings &cfg)
 			break;
 #	else
 		case Mode::Qt5:
+		case Mode::Qt6Winink:
+		case Mode::Qt6Wintab:
 			currentMode = Mode::Qt5;
-			inputMode = "Qt5 tablet input";
 			break;
 #	endif
-		default:
+		case Mode::Uninitialized:
 			break;
 		}
-	}
+	});
 #else
 	// Nothing to do on other platforms.
 	Q_UNUSED(app);
-	Q_UNUSED(cfg);
 #endif
 }
 
 QString current()
 {
-	return QString::fromUtf8(inputMode);
+	switch (currentMode) {
```

Comment:
It sounds like that should probably be transmitting just the raw enumeration value instead of a full string. For now I have modified this change so this function now returns the translation source string literal instead of `QString`, and then translates in the About window.

FWIW, anything i18n-related I have been pretty lazy about it in this patch because I know all of those need to be touched again soon when the dynamic translations stuff gets reworked and added.

Reply:
Yeah, that would make more sense, now that that enum actually exists.